core/templates/dev/head/pages/state_editor/StateSolutionEditorDirective.js

```diff
@@ -57,6 +57,16 @@ oppia.directive('stateSolutionEditor', [
           $scope.StateInteractionIdService = StateInteractionIdService;
           $scope.StateSolutionService = StateSolutionService;
 
+          $scope.getTooltip = function() {
+            if (StateEditorService.isInQuestionMode()) {
+              return 'There is no correct response that corresponds to ' +
+                'this solution. Check that there\'s a response with this ' +
+                'solution answer, and that response is marked as correct';
```

Comment:
solution answer -> solution as the answer

Also, the solution just has to match the answer group rule right? Not sure if that is too clear from the message. 

Maybe something along the lines of:
This solution doesn't correspond to a response marked as correct. Verify the responses specified, or change the solution.

Reply:
Also, missing . at the end.

Reply:
Done